app/Nova/User.php

```diff
@@ -132,6 +133,23 @@ public function fields(Request $request): array
                 ->creationRules('unique:users,github_username')
                 ->updateRules('unique:users,github_username,{{resourceId}}'),
 
+            File::make('Resume', function () {
+                return $this->resume_date ? 'resumes/'.$this->uid.'.pdf' : null;
+            })->path('resumes')
+                ->disk('local')
+                ->deletable(false)
```

Comment:
How exactly are we supposed to delete resumes

Reply:
There isn’t a way to delete them currently other than `rm` and a database edit.